src/frontend/Komponenter/Journalføring/KlageGjelderTilbakekreving.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import { Checkbox } from '@navikt/ds-react';
+
+interface KlageGjelderTilbakekrevingProps {
```

Comment:
```suggestion
interface Props {
```
Jeg synes koden blir litt kortere og greiere med dette navnet. Det er vel implisitt at `Props` tilhører `KlageGjelderTilbakekreving`?